theme/documentation/permissions.md

```diff
@@ -8,19 +8,22 @@ Here is a breakdown of the user tags:
 
 - A limited support user, an employee, a customer, a partner, or a watcher, will have the **KB** tag.
 - Trial users will not have the **KB** tag.
-- The **KB** tag will allow the user to view the full content of Knowledge Base Articles, product landing pages, the mega menu, announcements, quick links on the homepage, and CTAs (calls to actions).
+- The **KB** tag will allow the user to view the full content of Knowledge Base Articles, all tabs of the product landing pages, the mega menu, announcements, quick links on the homepage, and CTAs (calls to actions). Note: Official Documentation i.e. the Documentation tab on product landing pages, is open to all users, the KB tag gives access to everything else.
```

Comment:
Should we wrap the `ie` in parentheses?

Note: Official Documentation (i.e. the Documentation tab on product landing pages), is open to all users; the KB tag gives access to everything else.